cmake/modules/extensions.cmake

```diff
@@ -1559,6 +1559,66 @@ function(zephyr_syscall_header_ifdef feature_toggle)
   endif()
 endfunction()
 
+# Verify blobs fetched using west. If the sha256 checksum isn't valid, a warning/
+# fatal error message is printed (depends on REQUIRED flag).
+# Paths are relative to the current module's zephyr/blobs directory.
+#
+# Usage:
+#   zephyr_blobs_verify([PATHS path [paths...]]
```

Comment:
please let the caller decide between `MODULES` or `PATHS`.

Also, imo `FILES` are a better name than `PATHS`.

So that we have:
```
zephyr_blobs_verify(<MODULE <name>|FILES file [file...]> [REQUIRED])
```

Reply:
It is possible to have the same relative path (to module's `zephyr/blobs`') for different modules, which can be a bit tricky if only files are listed?

Should `MODULE` become a required argument, now that `ZEPHYR_CURRENT_MODULE_NAME` is available?

Reply:
> It is possible to have the same relative path (to module's zephyr/blobs') for different modules, which can be a bit tricky if only files are listed?

which was exactly the reason to support abspath.
A caller outside of a module's cmake code can then use: `${ZEPHYR_<module>_MODULE_DIR}/path/to/file` to set abspath.
A caller inside the module can use a path relative to the current CMakeLists.txt file (which sets CMAKE_CURRENT_SOURCE_DIR when added).

> Should MODULE become a required argument, now that ZEPHYR_CURRENT_MODULE_NAME is available?

No, with abspath support, the module is not needed as required arg.